scripts/backup

```diff
@@ -13,12 +13,12 @@ set -eu
 # IMPORT GENERIC HELPERS
 #=================================================
 
-#if [ ! -e _common.sh ]; then
-#	# Get the _common.sh file if it's not in the current directory
-#	cp ../settings/scripts/_common.sh ./_common.sh
-#	chmod a+rx _common.sh
-#fi
-#source _common.sh
+if [ ! -e _common.sh ]; then
+	# Get the _common.sh file if it's not in the current directory
+	cp ../settings/scripts/_common.sh ./_common.sh
+	chmod a+rx _common.sh
+fi
```

Comment:
You don't to copy `_common.sh`.
You can just do that : `source ../settings/scripts/_common.sh`
Like here : https://github.com/YunoHost-Apps/synapse_ynh/blob/master/scripts/backup#L17